doc/guides/writing-tests.md

```diff
@@ -264,7 +264,8 @@ These imported tests will be wrapped like this:
 
 ```js
 /* eslint-disable */
-/* WPT Refs:
+/* The following tests are copied from WPT, modifications to them should be upstreamed first.
```

Comment:
Can you please keep the lines under 80 chars? Not sure why the linter didn't catch that.